pkg/controller/controller_binding.go

```diff
@@ -714,8 +714,11 @@ func (c *controller) reconcileServiceBinding(binding *v1beta1.ServiceBinding) er
 			}
 		} else {
 			if toUpdate.Status.CurrentOperation != v1beta1.ServiceBindingOperationUnbind {
-				// Cancel any pending orphan mitigation since the resource is being deleted
-				toUpdate.Status.OrphanMitigationInProgress = false
+				// Cancel any in-progress operation or pending orphan mitigation since the resource is being deleted.
```

Comment:
Looks like this will have to merge with https://github.com/kubernetes-incubator/service-catalog/pull/1544

Noted.